arch/x86/core/intel64/locore.S

```diff
@@ -39,7 +39,9 @@
 	movl %eax, %cr4
 	clts
 
-	/* Page tables created at build time by gen_mmu.py */
+	/* Page tables created at build time by gen_mmu.py
+	 * NOTE: Presumes phys=virt
```

Comment:
NOTE: I am terrible at x64 assembly, there are weird rules about relocations and immediate values. While I'm focused on paging I at least left a note, but if someone knows how to do this feel free to share.